pkg/controllers/testutil/zcmd/zfs/get.go

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
I think this test_util is much specific to cspi controller. 
Why are we putting it at `pkg/controller` level ? How about putting at `pkg/pool` pr near to zcmd ? Are there any import loops that can happen ? Let us discuss this.

Reply:
> Why are we putting it at pkg/controller level ? 

Not to spread test-related code in the codebase.

Reply:
Let's discuss on this

Reply:
As discussed if it is really required to move out then we can do that.